### ISSUE TITLE
SearchVenuesActivity no results are empty

### ISSUE THREAD

**Issue by GoogleCodeExporter** [iamakshayv/foursquared#122]
```
Search for a venue that doesn't exist. You'll end up with a black or white 
screen without a "Venues not found" notice.
```

Original issue reported on code.google.com by `jlapenna` on 3 Nov 2009 at 12:35

**Comment by GoogleCodeExporter** [comment]
```
I've attached a fix for this problem. It seemed really easy to fix this, so 
easy that I 
suspect there is something I've missed.
```

Original comment by `brampton` on 6 Dec 2009 at 12:25

Attachments:
- [no_search_results.diff](https://storage.googleapis.com/google-code-attachments/foursquared/issue-122/comment-1/no_search_results.diff)

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 39d78ce33a.
```

Original comment by `jlapenna` on 7 Dec 2009 at 12:30
- Changed state: **Fixed**
